src/components/ProofOfAddress/PoAGuidance/index.tsx

```diff
@@ -1,23 +1,21 @@
-import { h } from 'preact'
```

Comment:
Same here. This SDK component is likely to crash the SDK with a ReferenceError when this screen needs to be rendered for the PoA flow without the `import { h } from 'preact'`

Reply:
This is here: import { FunctionComponent, h } from 'preact'